service/raft/group0_state_machine.cc

```diff
@@ -83,6 +84,14 @@ static mutation extract_history_mutation(std::vector<canonical_mutation>& muts,
     return res;
 }
 
+static future<> mutate_locally(utils::chunked_vector<canonical_mutation> muts, storage_proxy& sp) {
+    auto db = sp.data_dictionary();
+    co_await max_concurrent_for_each(muts, 128, [&sp, &db] (const canonical_mutation& cmut) -> future<> {
+        auto schema = db.find_schema(cmut.column_family_id());
+        co_await sp.mutate_locally(cmut.to_mutation(schema), nullptr, db::commitlog::force_sync::yes);
```

Comment:
nit: pessimizing coroutine, could be `return` (nothing happens after the `co_await`)

Reply:
ok